### PR TITLE
add model

### PR DESCRIPTION
**Pull request by longnv-99**
## Related Tickets:
- [#40791] https://edu-redmine.sun-asterisk.vn/issues/40791 

## WHAT this PR does?
- init model

## Notes Impacted Areas:
- N/A.

## Attachments:
db

### CODE REVIEW

File: app/Models/Book.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Book extends Model
+{
+    use HasFactory;
+
```
Comment (hungsondo) [inline app/Models/Book.php:11]: Nếu ô định dùng soft delete thì cần Use Softdelete

Comment (longnv-99) [reply app/Models/Book.php:11]: ok

**Comment by truongcr7** [comment]
ok

**Comment by hungsondo** [comment]
ok

**Comment by longnv-99** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**